packages/integration-karma/test/component/LightningElement.errorCallback/x/boundaryChildRenderedThrowFrozen/boundaryChildRenderedThrowFrozen.js

```diff
@@ -0,0 +1,8 @@
+import { LightningElement } from 'lwc';
+
+export default class ChildRenderedThrow extends LightningElement {
+    didError = false;
+    errorCallback() {
+        this.didError = true;
```

Comment:
Capture the error argument of errorCallback() in a public property and assert the error message in the test.